### PR TITLE
TE-1725 Expose a CookieAlert component

### PR DESCRIPTION
**Pull request by red-runner**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-1725)

### What **one** thing does this PR do?
The `Modal`  exposes props which allow for the behaviours show in the ticket; then, implements a new `CookieAlert` component with the required styles.

### Any other notes
kapture 2019-01-29 at 12 46 19

### CODE REVIEW

File: src/components/general-widgets/CookieAlert/Readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return (      
+      <CookieAlert
+        text='Led through the mist, by the milk-light of moon, all that was lost is revealed. Our long bygone burdens, mere echoes of the spring, but where have where we come and where shall we end? If dreams cant come true, then why not pretend... How the gentle wind, beckons through the leaves, as Autumn, colors, fall. Then sing in a swirl, of golden memories, the loveliest, lies, of all.'
+        buttonText="Acceptance"
+        isOpen={this.state.isOpen}
+        onClick={() => {alert('Potatoes and Molasses')}}
+        trigger={
+          <Button isRounded onClick={this.handleChange}>Trigger Cookie</Button>
+        }
+        onClose={this.handleChange}
```
Comment (marcdavi-es) [inline src/components/general-widgets/CookieAlert/Readme.md]: For the purposes of the documentation, I think it's less noisy to...

1. Remove `handleChange`
2. Inline here

```js
onClose={() => this.setState({ isOpen: false })}
```

File: src/components/elements/Modal/component.js [hunk 2]
```diff
@@ -40,13 +57,23 @@ Component.defaultProps = {
...
+  /** Does the modal have a margin. */
+  hasMargin: PropTypes.bool,
+  /** Is the modal aligned bottom. */
+  isAlignedBottom: PropTypes.bool,
   /** Is the modal filling the whole screen when displayed. */
   isFullscreen: PropTypes.bool,
+  /** Is the modal justified right.*/
+  isJustifiedRight: PropTypes.bool,
+  /** Are the edges of the modal round */
+  isModalRounded: PropTypes.bool,
```
Comment (marcdavi-es) [inline src/components/elements/Modal/component.js]: `hasRoundedCorners` to be consistent with e.g. `Thumbnail`, `ResponsiveImage`

File: src/components/general-widgets/CookieAlert/component.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  onClick,
+  onClose,
+  text,
+  trigger,
+}) => (
+  <Modal
+    dimmer
+    hasMargin
+    isAlignedBottom
+    isBackgroundTransparent
```
Comment (marcdavi-es) [inline src/components/general-widgets/CookieAlert/component.js]: What does this prop do?

Comment (red-runner) [reply src/components/general-widgets/CookieAlert/component.js]: @marcdavi-es This was accidentally left behind

**Pull request closed without merging**